pkg/constant/config.go

```diff
@@ -77,4 +77,5 @@ const (
 
 const (
 	FeatureGateIgnoreConfigTemplateDefaultMode = "IGNORE_CONFIG_TEMPLATE_DEFAULT_MODE"
+	FeatureGateComponentReplicasAnnotation     = "COMPONENT_REPLICAS_ANNOTATION"
```

Comment:
it should  be defined in the file viper_config.go? config.go is specifically used for defining constants about configuration module.

Reply:
fixed by [660ca13](https://github.com/apecloud/kubeblocks/pull/7902/commits/660ca13f04e36567c338a89a1e4900c604e7afd8)